java/vector/src/test/java/org/apache/arrow/vector/types/pojo/TestExtensionType.java

```diff
@@ -85,21 +88,21 @@ public void roundtripUuid() throws IOException {
           final ArrowFileReader reader = new ArrowFileReader(channel, allocator)) {
         reader.loadNextBatch();
         final VectorSchemaRoot readerRoot = reader.getVectorSchemaRoot();
-        Assert.assertEquals(root.getSchema(), readerRoot.getSchema());
+        assertEquals(root.getSchema(), readerRoot.getSchema());
```

Comment:
Just a thought: since we are updating test cases, shall we use JUnit5? `juputer.api.`?